auth/oauth.go

```diff
@@ -26,13 +26,13 @@ import (
 	"runtime"
 	"strings"
 
-	atlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas-sdk/core"
 )
 
-const defaultBaseURL = atlas.CloudURL
+const defaultBaseURL = "https://cloud.mongodb.com/"
 
 var (
-	userAgent = fmt.Sprintf("go-mongodbatlas/%s (%s;%s)", atlas.Version, runtime.GOOS, runtime.GOARCH)
+	userAgent = fmt.Sprintf("go-mongodbatlas/%s (%s;%s)", core.Version, runtime.GOOS, runtime.GOARCH)
```

Comment:
We likely need to change this to go-sdk-auth to avoid confusion when we release 0.1.0.